### ISSUE TITLE
On Session Restore Page in Firefox, NVDA fails to turn on focus mode for the tab/window tree

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3376]
**Reported by ppatel on 2013-07-20 17:45**
Upon Firefox crashing or a few other circumstances, the user encounters the restore session page which allows the user to close certain tabs that could be causing problems. With the current NVDA snapshot, NVDA does not allow the user to focus on the tree control for the tab selection by pressing enther the enter key or the space key. The user is forced to turn on focus mode manually by invoking NVDA+space keys.
# Is this behavior specific to this page?
# Are other tree controls treated in the same manner?

**Comment by derekriemer** [comment]
closing as worksfome
